src/code/z_actor.c

```diff
@@ -457,439 +1231,3485 @@ f32 Actor_XZDistanceToPoint(Actor* actor, Vec3f* point) {
+    gSPEndDisplayList(displayListHead++);
+
+    return displayList;
+}
+
+// unused
+void func_800BD9E0(GlobalContext* globalCtx, SkelAnime* skelAnime, OverrideLimbDraw overrideLimbDraw,
+                   PostLimbDraw postLimbDraw, Actor* actor, s16 alpha) {
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+    func_8012C28C(globalCtx->state.gfxCtx);
+
+    gDPSetEnvColor(POLY_OPA_DISP++, 0, 0, 0, alpha);
+    gSPSegment(POLY_OPA_DISP++, 0x0C, gEmptyDL);
+
+    POLY_OPA_DISP = SkelAnime_DrawFlex(globalCtx, skelAnime->skeleton, skelAnime->jointTable, skelAnime->dListCount,
+                                       overrideLimbDraw, postLimbDraw, actor, POLY_OPA_DISP);
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void func_800BDAA0(GlobalContext* globalCtx, SkelAnime* skelAnime, OverrideLimbDraw overrideLimbDraw,
+                   PostLimbDraw postLimbDraw, Actor* actor, s16 alpha) {
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+    func_8012C2DC(globalCtx->state.gfxCtx);
+
+    gDPSetEnvColor(POLY_XLU_DISP++, 0, 0, 0, alpha);
+    gSPSegment(POLY_XLU_DISP++, 0x0C, func_800BD9A0(globalCtx->state.gfxCtx));
+
+    POLY_XLU_DISP = SkelAnime_DrawFlex(globalCtx, skelAnime->skeleton, skelAnime->jointTable, skelAnime->dListCount,
+                                       overrideLimbDraw, postLimbDraw, actor, POLY_XLU_DISP);
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+// Unused
+s16 func_800BDB6C(Actor* actor, GlobalContext* globalCtx, s16 arg2, f32 arg3) {
+    Player* player = GET_PLAYER(globalCtx);
+    f32 phi_f2;
+
+    if ((globalCtx->csCtx.state != 0) || (D_801D0D50 != 0)) {
+        phi_f2 = Math_Vec3f_DistXYZ(&actor->world.pos, &globalCtx->view.eye) * 0.25f;
+    } else {
+        phi_f2 = Math_Vec3f_DistXYZ(&actor->world.pos, &player->actor.world.pos);
+    }
+
+    if (arg3 < phi_f2) {
+        actor->flags &= ~ACTOR_FLAG_1;
+        Math_SmoothStepToS(&arg2, 0, 6, 0x14, 1);
+    } else {
+        actor->flags |= ACTOR_FLAG_1;
+        Math_SmoothStepToS(&arg2, 0xFF, 6, 0x14, 1);
+    }
+
+    return arg2;
+}
+
+void func_800BDC5C(SkelAnime* skelAnime, ActorAnimationEntry* animation, s32 index) {
+    f32 frameCount;
+
+    animation += index;
+    if (animation->frameCount > 0.0f) {
+        frameCount = animation->frameCount;
+    } else {
+        frameCount = Animation_GetLastFrame(&animation->animation->common);
+    }
+
+    Animation_Change(skelAnime, animation->animation, animation->playSpeed, animation->startFrame, frameCount,
+                     animation->mode, animation->morphFrames);
+}
+
+// Unused
+void func_800BDCF4(GlobalContext* globalCtx, s16* arg1, s16* arg2, s32 size) {
+    s32 frames = globalCtx->gameplayFrames;
+    s32 i;
+
+    for (i = 0; i < size; i++) {
+        arg1[i] = (0x814 + 50 * i) * frames;
+        arg2[i] = (0x940 + 50 * i) * frames;
+    }
+}
+
+void Actor_Noop(Actor* actor, GlobalContext* globalCtx) {
+}
+
+#include "z_cheap_proc.c"
```

Comment:
Why is this a separate file?

Reply:
Debugging strings from OoT and MM showed those 2 functions are in a different file